modules/file_storage/main.tf

```diff
@@ -16,32 +12,27 @@ resource "google_storage_bucket" "file_storage" {
   labels = var.labels
 
   cors {
+    origin          = ["*"]
```

Comment:
It would be more secure to limit this to the urls the service can be rendered from, but it's a pretty low severity issue as all objects will be private in the bucket anyway...